include/openssl/evp.h

```diff
@@ -1321,6 +1325,8 @@ void EVP_PKEY_asn1_set_security_bits(EVP_PKEY_ASN1_METHOD *ameth,
  * Method handles all operations: don't assume any digest related defaults.
  */
 # define EVP_PKEY_FLAG_SIGCTX_CUSTOM     4
+/* Do a customized hashing process */
+# define EVP_PKEY_FLAG_DIGEST_CUSTOM     8
```

Comment:
Similarly this seems misplaced.

Reply:
Hmm, if this is placed in an internal header file, then the value of the flag should be chosen safe...

Reply:
I'd suggest a comment in the public header stating that the value 8 is reserved for internal use. In the internal header file where we put the definition we add a reference to the public header values.

Reply:
Why is this flag even needed?  If there's a non-NULL `digest_custom` in a `EVP_PKEY_METHOD`, then it has a digest custom function...  isn't that enough?
(I know that we have sometimes used flags to tell if certain pointers are defined or not, but *that's from the time when all structures were public*, so flagging was done to allow for a certain level of ABI backward compatibility, to tell if added fields could be relied upon.  Since `EVP_PKEY_METHOD` is *opaque*, this methodology is no longer needed)